services/ai-navigator-app/0.2.3-d2iq.0/defaults/cm.yaml

```diff
@@ -56,7 +56,7 @@ data:
         priorityClassName: dkp-high-priority
       enabled: true
       image:
-        tag: 15.2.0-debian-11-r21
+        tag: 15.2.0-debian-11-r21-d2iq.0
```

Comment:
@asekretenko, do we have to retain the same image tag or feasible to change to updated version ?